Modules/Core/Mesh/test/itkTriangleMeshCurvatureCalculatorTest.cxx

```diff
@@ -95,7 +95,7 @@ itkTriangleMeshCurvatureCalculatorTest(int argc, char * argv[])
   curvCalculator->SetCurvatureTypeToGaussian();
   curvCalculator->Compute();
 
-  gaussCurvatureData = curvCalculator->GetGaussCurvatureData();
+  gaussCurvatureData = curvCalculator->GetModifiableGaussCurvatureData();
```

Comment:
@PranjalSahu Please consider having two separate commits: one for the `Wmaybe-uninitialized` warning, and one for the `GetModifiable` calls. (Following the rule "One Commit. One Change".)

Reply:
Yes will make two separate PRs.